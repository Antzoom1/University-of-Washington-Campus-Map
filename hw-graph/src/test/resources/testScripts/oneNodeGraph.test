# Tests adding one node to the graph

# Create a graph
CreateGraph oneNodeGraph

# Add two nodes
AddNode oneNodeGraph 1

# Print the node in the graph, the outgoing edges from 1
ListNodes oneNodeGraph
ListChildren oneNodeGraph 1
