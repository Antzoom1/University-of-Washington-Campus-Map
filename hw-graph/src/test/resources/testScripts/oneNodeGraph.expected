# Tests adding one node to the graph

# Create a graph
created graph oneNodeGraph

# Add two nodes
added node 1 to oneNodeGraph

# Print the node in the graph, the outgoing edges from 1
oneNodeGraph contains: 1
the children of 1 in oneNodeGraph are:
